service/cziti/sdk.go

```diff
@@ -656,68 +706,25 @@ func eventCB(ztx C.ziti_context, event *C.ziti_event_t) {
 				servicesToAdd = append(servicesToAdd, svcToAdd)
 			}
 		}
-		var minimumTimeout int32 = -1
```

Comment:
all this logic moved - but i don't see UpdateMFATime() called now? is that on purpose? seems like maybe a bug? please check

Reply:
i also didn't diff the changes. i am simply assuming you jsut moved all this logic?

Reply:
UpdateMFATime call is in line 711 now. I copied the whole logic to the UpdateMFATime  function . I didnt change it